src/ts/waterfall/sub-components/svg-marks.ts

```diff
@@ -22,6 +22,7 @@ export function createMarks(context: Context, marks: Mark[]) {
     let lineHolder = svg.newG("line-holder");
     let lineLabelHolder = svg.newG("line-label-holder");
     let lineLabel = svg.newTextEl(mark.name, { x: x + "%", y: diagramHeight + 25 });
+    lineLabel.setAttribute("writing-mode", "tb");
```

Comment:
fix for Safari, IE and Edge - so SVG support for `writing-mode: vertical-lr` yet